modules/basics/src/main/java/com/opengamma/strata/basics/currency/MultiCurrencyAmountArray.java

```diff
@@ -56,14 +56,16 @@
     implements FxConvertible<CurrencyAmountArray>, ImmutableBean, Serializable {
 
   /**
+   * The size of this array.
```

Comment:
What are the semantics of `size` supposed to be in this class? It looks like the previous semantics were that `size` was the number of currencies, i.e. consistent with `size` on `MultiCurrencyAmount`. Now it seems to be either this, or the number of `MultiCurrencyAmount` items (i.e. scenarios) that it was constructed from, depending on how it is constructed.